src/builders/dev-server/index.ts

```diff
@@ -24,6 +26,17 @@ export class SingleSpaDevServer extends DevServerBuilder {
         SINGLE-SPA-ANGULAR: Angular dev server is serving application as a single module`);
         return webpackConfig;
     }    
+    _buildServerConfig(root, projectRoot, options, browserOptions) {
+        // @ts-ignore
+        const devServerConfig = super._buildServerConfig(root, projectRoot, options, browserOptions);
+
+        return webpackMerge.smart(devServerConfig, {
+            // @ts-ignore
+            contentBase: path.resolve(root, projectRoot.serveDirectory || '../'),
+            historyApiFallback: true,
+            publicPath: root.slice(root.lastIndexOf(path.sep)) + '/' + options.outputPath,
```

Comment:
example: `/test1/dist/test1/`

This is where the webpack assets are served from.